### PR TITLE
Remove include from hh that is in cc

### PR DESCRIPTION
**Pull request by rraustad**
Pull request overview
---------------------
 - Fixes VS compiler squiggly warning
 - Removes anoying VS warning in HeatingCapacitySizing.cc. Likely more of this to come.
 - Also cleans out redundant code

image

**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc [hunk 1]
```diff
@@ -88,541 +90,172 @@ Real64 CoolingAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, b
...
-                            this->finalZoneSizing(this->curZoneEqNum).CoolDDNum <= state.dataEnvrn->TotDesDays) {
-                            DDNameFanPeak = state.dataWeatherManager->DesDayInput(this->finalZoneSizing(this->curZoneEqNum).CoolDDNum).Title;
-                            dateTimeFanPeak =
-                                format("{}/{} {}",
-                                       state.dataWeatherManager->DesDayInput(this->finalZoneSizing(this->curZoneEqNum).CoolDDNum).Month,
-                                       state.dataWeatherManager->DesDayInput(this->finalZoneSizing(this->curZoneEqNum).CoolDDNum).DayOfMonth,
-                                       state.dataRptCoilSelection->coilSelectionReportObj->getTimeText(
-                                           state, this->finalZoneSizing(this->curZoneEqNum).TimeStepNumAtCoolMax));
-                        }
+                        coolingFlow = true;
```
Comment (rraustad) [inline src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc:93]: These strings are only reported for fan components. No need for 1) the redundancy, and 2) to writing these strings for other non-fan components. Moved these to bottom of function and replaced with a bool to know which string to report.

Comment (rraustad) [inline src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc:93]: These strings are only reported for fan components. No need for 1) the redundancy, and 2) to writing these strings for other non-fan components. Moved these to bottom of function and replaced with a bool to know which string to report.

File: src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc [hunk 2]
```diff
@@ -117,28 +117,14 @@ Real64 HeatingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
...
-                            DesMassFlow = this->finalZoneSizing(this->curZoneEqNum).DesHeatMassFlow;
-                        }
-                        CoilInTemp = this->setHeatCoilInletTempForZoneEqSizing(
-                            this->setOAFracForZoneEqSizing(state, DesMassFlow, zoneEqSizing(this->curZoneEqNum)),
-                            zoneEqSizing(this->curZoneEqNum),
-                            finalZoneSizing(this->curZoneEqNum));
-                        CoilInHumRat = this->setHeatCoilInletHumRatForZoneEqSizing(
-                            this->setOAFracForZoneEqSizing(state, DesMassFlow, zoneEqSizing(this->curZoneEqNum)),
-                            zoneEqSizing(this->curZoneEqNum),
-                            finalZoneSizing(this->curZoneEqNum));
```
Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc:134]: Duplicate code reported by clang-tidy. Deleting this section will allow to fall through to else where same code is executed..

Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc:134]: Duplicate code reported by clang-tidy. Deleting this section will allow to fall through to else where same code is executed..

File: src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc [hunk 3]
```diff
@@ -252,8 +238,6 @@ Real64 HeatingCapacitySizer::size(EnergyPlusData &state, Real64 _originalValue,
                             }
                         } else if (this->curDuctType == DataHVACGlobals::Heating) {
                             DesVolFlow = this->finalSysSizing(this->curSysNum).DesHeatVolFlow;
-                        } else if (this->curDuctType == DataHVACGlobals::Other) {
-                            DesVolFlow = this->finalSysSizing(this->curSysNum).DesMainVolFlow;
```
Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc:256]: Similar to above, duplicate code, fall through to else.

Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.cc:256]: Similar to above, duplicate code, fall through to else.

File: src/EnergyPlus/Autosizing/HeatingCapacitySizing.hh [hunk 4]
```diff
@@ -49,7 +49,6 @@
 #define HeatingCapacitySizing_hh_INCLUDED
 
 #include <EnergyPlus/Autosizing/BaseSizerWithScalableInputs.hh>
-#include <EnergyPlus/DataHVACGlobals.hh>
```
Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.hh:52]: Duplicate include seemed to cause Intellisense to report "incomplete type not allowed" (see issue description)

Comment (rraustad) [inline src/EnergyPlus/Autosizing/HeatingCapacitySizing.hh:52]: Duplicate include seemed to cause Intellisense to report "incomplete type not allowed" (see issue description)

**Comment by rraustad** [comment]
This is ready for review and merge.

**Comment by Myoldmopar** [comment]
Yeah this seems super straightforward, and CI is all happy, essentially confirming you haven't broken anything with these changes.  I'll do a quick build/test here in a bit or first thing in the morning and we'll get this merged.

**Comment by Myoldmopar** [comment]
All happy locally, this is going in.  Thanks @rraustad !

**Pull request merged**